src/hooks/useSelectionLogic.ts

```diff
@@ -78,9 +78,6 @@ export function useSelectionLogic<T extends HTMLElement>({
     if (onChangeRefId.current) {
       cancelAnimationFrame(onChangeRefId.current);
     }
-    if (currentSelectionEnd?.current) {
-      currentSelectionEnd.current();
-    }
```

Comment:
@tomaszczura I didn't think it made sense to fire the `onSelectionEnd` here because [there is another time it fires (which I don't understand when that would happen](https://github.com/AirLabsTeam/react-drag-to-select/pull/37/files#diff-1fa8fd0fedac2a98b4ebd3ef62cab7e2de5b963eb6472c533b6c4d3c69c1621eL138) and I wanted to be sure the event only fired for the consumer when they were actually dragging.